app/code/community/Mpchadwick/PageCacheHitRate/Model/Observer.php

```diff
@@ -40,6 +41,20 @@ public function handleControllerFrontSendResponseBefore(Varien_Event_Observer $o
         }
     }
 
+    /**
+     * Determine this request is cacheable in FPC
+     *
+     * @return bool
+     */
+    protected function isRequestCacheable()
+    {
+        $request = Mage::app()->getRequest();
+        $processor = Mage::getSingleton('enterprise_pagecache/processor');
+        $subprocessor = $processor->getMetadata('cache_subprocessor');
+
+        return $subprocessor !== null && $processor->canProcessRequest($request);
+    }
```

Comment:
This code will only execute for cache misses or partials meaning that the cacheable param will not be added for hits. For hits the observer won't fire and the tracking is handled by `Mpchadwick_PageCacheHitRate_Model_Processor`.

Both consult `Mpchadwick_PageCacheHitRate_Model_Tracker_ParamProvider::baseParams` which is probably where this value should come from. That way you only need to write the logic once.